torch/fx/_symbolic_trace.py

```diff
@@ -250,6 +250,13 @@ def __init__(
         self.param_shapes_constant = param_shapes_constant
 
         self.submodule_paths: Optional[Dict[torch.nn.Module, str]] = None
+        self.root_module_name: str = ""
+        # Maps the containing module's name to the operator name
+        self.scope = Scope("", None)
+        # Records the module call stack
+        self.module_stack = collections.OrderedDict()
+        # Mapping of node name to module scope
+        self.node_name_to_scope: Dict[str, Tuple[str, type]] = {}
```

Comment:
another thing we could do is just to attach the scope to node itself, but this will involve quantization code as well and we can do it in a separate PR

Reply:
ok that sounds good.